src/sentry/static/sentry/app/views/settings/projectAlerts/ruleRow.tsx

```diff
@@ -52,7 +53,7 @@ class RuleRow extends React.Component<Props, State> {
     });
 
     const environmentName = data.environment
-      ? getDisplayName({name: data.environment})
+      ? getDisplayName({name: data.environment} as Environment)
```

Comment:
are you sure about this casting? The type Environment is not of type string

Reply:
`data` is typed as `IssueAlertRule | SavedIncidentRule` and will `data.environment` will resolve into `null | string`. It's not the best but it'll work for `Environment['name']` without making major changes to this component.

Reply:
I see... in this case I would cast it as: `as Environment['displayName']` or  `as Environment['name']` because both are of type string

Reply:
You're right about typecasting being odd here. The problem is the type on `getDisplayName` and I've fixed that.